apps/desktop/src/components/views/terminal/terminal-panel.tsx

```
@@ -0,0 +1,430 @@
+            </span>
+          )}
+          {/* Font size indicator */}
+          {fontSize !== DEFAULT_FONT_SIZE && (
+            <button
+              onClick={(e) => {
+                e.stopPropagation();
+                resetZoom();
+              }}
+              className="rounded px-1 text-[10px] text-muted-foreground transition-colors hover:bg-accent hover:text-foreground"
+              title="Click to reset zoom (Ctrl+0)"
+            >
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)